src/riak_repl_pb_get.erl

```diff
@@ -51,13 +51,23 @@ decode(_, Bin) ->
     {ok, Bin}.
 
 %% @doc encode/1 callback. Encodes an outgoing response message.
-encode(#rpbgetresp{} = Msg) ->
+encode(Msg) ->
     {ok, riak_pb_codec:encode(Msg)}.
 
 %% Process Protocol Buffer Requests
 %%
 %% @doc Return Cluster Id of the local cluster
-process(#rpbreplgetclusteridreq{}, State) ->
+process(<<Code:8, Data/binary>>, State) ->
```

Comment:
The pattern elsewhere is to do the decode at the decode step not by a pattern match at process e.g. https://github.com/basho/riak_kv/blob/develop-3.0/src/riak_kv_pb_aaefold.erl#L65-L66 or https://github.com/basho/riak_kv/blob/develop-3.0/src/riak_kv_pb_object.erl#L84-L98

Can this be made consistent with these other implementations?

Reply:
I am decoding it here because the message to be processed comes via [`riakc_pb_socket:tunnel`](https://github.com/TI-Tokyo/riak_repl_pb_api/blob/fd96ef21e2c9ab0d686e5a08c7d8944e9a2eb647/src/riak_repl_pb_api.erl#L39), which is [not decoded on receipt](https://github.com/basho/riak-erlang-client/blob/develop-3.0/src/riakc_pb_socket.erl#L2047) in riakc, hence the inconsistency.